subprojects/packagefiles/cpr/meson.build

```diff
@@ -1,31 +1,40 @@
 project(
   'cpr',
   'cpp',
-  version: '1.12.0',
+  version: '1.13.0',
   license: 'MIT',
   default_options: 'cpp_std=c++17',
 )
```

Comment:
Let's also add `meson_version: '>=0.63.0'` while we're here.  (Alternatively `0.54.0` if you move `cpp_std` to `override_options` on the `library()` and `executable()` calls, preferably via an intermediate variable.)